_posts/2021-07-20-5.0.0-release.md

```diff
@@ -0,0 +1,100 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+<!--
+
+To use this template:
+
+* Update all "XX" values with the appropriate numbers (you can get the resolved issues and contributors count from `_release/5.0.0.md`)
+* Fill in the various sections below. Note that the audience is the broader user community, not Arrow developers, so please write clearly using terms they will understand and care about. Delete any sections that don't have any content (as in, there are no changes to announce)
+* Delete this introductory comment
+
+ -->
+
+
+The Apache Arrow team is pleased to announce the 5.0.0 release. This covers
+over XX months of development work and includes [**XX resolved issues**][1]
+from [**XX distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 4.0.0 release, Daniël Heres, Kazuaki Ishizaki, Dominik Moritz, and Weston Pace
+have been invited as committers to Arrow,
+and Benjamin Kietzman and David Li have joined the Project Management Committee
+(PMC). Thank you for all of your contributions!
+
+## Columnar Format Notes
+
+Official IANA Media types (MIME types) have been registered for Apache
+Arrow IPC protocol data, both [stream]({{ site.baseurl }}/docs/format/Columnar.html#ipc-streaming-format) and [file]({{ site.baseurl }}/docs/format/Columnar.html#ipc-file-format) variants:
+
+* https://www.iana.org/assignments/media-types/application/vnd.apache.arrow.stream
+* https://www.iana.org/assignments/media-types/application/vnd.apache.arrow.file
+
+We recommend ".arrow" as the IPC file format file extension and ".arrows" for the IPC streaming format file extension.
+
+## Arrow Flight RPC notes
+
+The Go implementation now supports custom metadata and middleware, and has been added to integration testing.
+
+In Python, some operations can now be interrupted via Control-C.
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
+## Java notes
+
+## JavaScript notes
+
+## Python notes
+
+## R notes
+
+In this release, we've more than doubled the number of functions you can call on Arrow Datasets inside `dplyr::filter()` and `mutate()`, including many more string, datetime, and math functions. You can also write Datasets to CSV files, in addition to Parquet and Feather. We've also deepened support for the Arrow C interface, which is used in the Python interface and allows integration with other projects, such as DuckDB.
+
+For more on what’s in the 5.0.0 R package, see the [R changelog][4].
+
+## Ruby and C GLib notes
+
+### Ruby
+
+### C GLib
+
+## Rust notes
+
```

Comment:
Link here to the Rust release blog post (in the works at https://github.com/apache/arrow-site/pull/128) and changelog (https://github.com/apache/arrow-rs/blob/5.0.0/CHANGELOG.md)

Reply:
I don't think there is any way to know for sure what the URL of the Rust release blog post will be, and it might be posted after this one, so I'll just refer to it by name without a link for now.